share/extender/build_input.yml

```diff
@@ -232,7 +232,7 @@ platforms:
         dynamicLibs: []
         symbols:    ["AudioDecoderTremolo"]
         osMinVersion:   "{{env.IOS_VERSION_MIN}}"
-        swiftFlags: ["-target", "arm64-apple-ios11.0"]
```

Comment:
Some pods require a higher min version than 11 to build

Reply:
It's interesting because we also hardcode it in the Info.plist and users that wish to change min version need to copy the Info.plist, modify the min version and point to it from game.project. Would it not make sense to perhaps make this value configurable from game.project in the iOS section, and also send this value to the extender server?

Reply:
I agree configurability is good.
Not sure about the location. Is game.project enough? Won't it need to be rebuilt? If so, then an app manifest makes more sense perhaps?

Reply:
The reason I suggested game.project was because that is where we configure similar version numbers for other platforms, such as Android min and target versions. And it also means that it will be easy to parameterize Info.plist:

https://github.com/defold/defold/blob/dev/engine/engine/content/builtins/manifests/ios/Info.plist#L81

I suppose we can pull it from game.project when we send the build request to the extender?